### PR TITLE
replace MYR into RM

### PR DESCRIPTION
**Pull request by usamabutt101**
### Why
Replace MYR into RM
Please give a short description why these code changes are being made, by replacing this line.

Link to Jira ticket: <place here>

### ToDo

- [ ]
- [ ]
- [ ]

If this is a `Draft:` or `WIP:` PR, please describe still needed actions, else delete section

### CODE REVIEW

File: src/components/adb-transaction-history/index.tsx [hunk 1]
```diff
@@ -373,7 +373,7 @@ const ADBTransactionHistoryComponent = (prop: ADBTransactionHistoryProps) => {
                         "Savings Pot Balance"}
                   </Text>
                   <Text style={styles.accountBalanceItemValue}>{`${
-                    item.currencyCode
+                    item.currencyCode === "RM" ? item.currencyCode : "RM"
```
Comment (sharmal33) [inline src/components/adb-transaction-history/index.tsx:376]: create a const value in aeon app(helpers/screen-utils.ts) and pass it as a props into this component. set default value to RM

File: src/components/adb-transaction-history/index.tsx [hunk 2]
```diff
@@ -79,7 +79,7 @@ export enum FilterType {
 
 const ADBTransactionHistoryComponent = (prop: ADBTransactionHistoryProps) => {
   const { colors, i18n } = useContext(ThemeContext);
-  const { wallets } = useContext(LoanOriginationContext);
+  const { wallets, financialData } = useContext(LoanOriginationContext);
   console.log("wallets", wallets);
```
Comment (sharmal33) [inline src/components/adb-transaction-history/index.tsx:83]: remove this log

**Review by sharmal33 — CHANGES_REQUESTED** [review]

File: src/components/adb-transaction-history/index.tsx (hunk 2 above)
Comment (usamabutt101) [reply src/components/adb-transaction-history/index.tsx:83]: log removed!

File: src/components/adb-transaction-history/index.tsx (hunk 1 above)
Comment (usamabutt101) [reply src/components/adb-transaction-history/index.tsx:376]: @sharmal33 actually the Currency code is coming from backend and earlier we were showing the code "MYR" which is coming from the backend so the backend didn't update this Currency code on their side, once they will fix then we will start using it that's why I put this temp solution for now.

Comment (usamabutt101) [reply src/components/adb-transaction-history/index.tsx]: I have set a const in (helpers/screen-utils.ts) as per your suggestion

**Review by sharmal33 — APPROVED** [review]

**Pull request merged**